qt/python/mantidqt/mantidqt/project/test/test_plotssaver.py

```diff
@@ -170,7 +170,7 @@ def test_get_dict_from_fig(self):
         return_value = self.plot_saver.get_dict_from_fig(self.fig)
 
         self.loader_plot_dict[u'creationArguments'] = [[{u"specNum": 2, "function": "plot", u"normalize_by_bin_width":
-                                                                                            True}]]
+                                                                                            False}]]
```

Comment:
I'm not clear on why this value changed. It seems that the sample workspace changed whether or not it was normalised to bin width when I added the lines to `plotssaver` and `plotsloader`. Some kind of undefined behaviour causing some unexpected consequence maybe?